api/__init__.py

```diff
@@ -0,0 +1,92 @@
+"""Backend api server that is used in the MOH project.
+
+A simple flask backend that will maintaine a live queue of current students in office hours and keep
+track of current student status in the within the queue
+by making request updating the `vist` table within the external postgrest database.
+"""
+
+# TODO: Make the doc string sound better
+
+
+import datetime
+from flask import Flask, request
+
+
+def create_app():
+    """Create and return flask api server
+
+    This funciton is used to setup the flask api server loading all it's depended modulars
+    """
+    app = Flask(__name__)
+
+    @app.route("/login", methods=["POST"])
+    def login():
+        """Checks if the current user have the right credentials to login
+        Args:
+            email: forum data field of email
+            password: forum data field of password
+
+        Returns:
+            The status of the login attempt
+        """
+        return "Login arrived"
+
+    @app.route("/signup", method=["POST"])
+    def signup():
+        """Creates an account using the given credentials, 
+        fails if email already register for an account
+        Args:
+            email: forum data field of email
+            password: forum data field of password
+
+        Returns:
+            The status of the signup attempt
+        """
+        return "Signup arrived"
+
+    @app.route("/health", methods=["GET"])
+    def health():
+        """Current health of the api server with meta data of the time"""
+        # Debug only, might write a debug wrapper later
+        return f"<p>Timestamp: {datetime.datetime.now().timestamp} | Server is healthy.</p>"
+
+    @app.route("/enqueue", method=["POST"])
+    def enqueue():
+        """Adding students into the current live queue of the office hour
```

Comment:
```suggestion
        """Add students to the current live queue for office hours
```

Does this add multiple _students_? Or just enqueue the current user? Let's be more precise.

Reply:
Add a single student, I think I was debating on if I should use web-socket on the queue status